requirements.txt

```diff
@@ -4,4 +4,5 @@ urllib3>=1.24.2
 pyxdg>=0.26
 sphinx
 wheel
+typing_extensions; python_version < '3.8'
```

Comment:
@jwhonce help me read this line, does this say: `Explicitly require typing_extensions for any python less than 3.8?`

If that is the case, we still need to include typing-extensions in rhel8.